release/release-notes/data-prepper.release-notes-2.2.0.md

```diff
@@ -0,0 +1,55 @@
+## 2023-04-20 Version 2.2.0
+
+---
+
+### Features
+* Support End-to-End Acknowledgments for S3 source ([#2394](https://github.com/opensearch-project/data-prepper/pull/2394), [#2426](https://github.com/opensearch-project/data-prepper/pull/2426), [#2436](https://github.com/opensearch-project/data-prepper/pull/2436), [#2458](https://github.com/opensearch-project/data-prepper/pull/2458), [#2465](https://github.com/opensearch-project/data-prepper/pull/2465))
+* Support S3 DLQ for OpenSearch sink ([#2298](https://github.com/opensearch-project/data-prepper/pull/2298))
+* Support Amazon Opensearch Serverless Sink ([#2169](https://github.com/opensearch-project/data-prepper/pull/2169))
+* Support S3 select in S3 source ([#1971](https://github.com/opensearch-project/data-prepper/pull/1971))
+* Support new list_to_map processor ([#2410](https://github.com/opensearch-project/data-prepper/pull/2410))
+* Support Snappy compression in S3 source ([#2420](https://github.com/opensearch-project/data-prepper/pull/2420))
+* Add append aggregation action for aggregate processor ([#2230](https://github.com/opensearch-project/data-prepper/pull/2230))
+
+### Enhancements
```

Comment:
Missing this one: 

Support format string option in add_entries processor #2464

Reply:
Good catch.